src/main/config/config.c

```diff
@@ -418,6 +419,32 @@ static void resetConf(void)
     featureSet(DEFAULT_FEATURES);
 #endif
 
+#ifdef SIRINFPV
+    featureSet(FEATURE_OSD);
+    featureSet(FEATURE_RX_SERIAL);
+    masterConfig.serialConfig.portConfigs[2].functionMask = FUNCTION_RX_SERIAL;
+    //masterConfig.batteryConfig.vbatscale = 20;
+    masterConfig.mag_hardware = MAG_NONE;            // disabled by default
+    masterConfig.rxConfig.serialrx_provider = SERIALRX_SBUS;
+    masterConfig.blackbox_device = 1;
+    masterConfig.blackbox_rate_num = 1;
+    masterConfig.blackbox_rate_denom = 1;
+#endif
+
+#ifdef OSD
+    masterConfig.vtx_channel = 19;
```

Comment:
We should try and avoid adding any target specific `#ifdef`s. The features can be set in the `target.h` file.